cluster-provision/images/vm-image-builder/build-containerdisk.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE[0]")"
+  pwd
+)"
+
+function cleanup() {
+  echo "[build-container-disk] cleanup"
+  rm -f Dockerfile
+
+  if [ $? -ne 0 ]; then
+    rm -f "$tar_file"
+    docker image rm "$full_image_tag"
+  fi
+}
+
+image_name=$1
+tag=$2
```

Comment:
Why a tag is needed? Isn't there a default? (latest)

Reply:
To enable users to pick their own tag

Reply:
Lets continue the "why"... Why would the users pick their own tag?
I fail to understand the scenario.

Reply:
You could create few container-disk images and test them instead of overriding the default `latest` tag, for example: 
creating container-disk with the name "fedora" and the tag will be "32"
then you could test this image by creating a VM and use it:
on VM yaml file:
```...
  image: localhost:<port>/fedora:32
...```
You could create another version of the image without loosing `fedora:32` image, or changing its tag manually
let say we created another version with the tag `fedora:tests`, you could use it in a VM like so:
```...
  image: localhost:<port>/fedora:tests
```